sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -446,10 +444,12 @@ def set_queue_access_policy(self,
     @distributed_trace
     def send_message(
             self,
-            content,  # type: Any
-            **kwargs  # type: Any
-    ):
-        # type: (...) -> QueueMessage
+            content: Any,
+            *,
+            visibility_timeout: Optional[int] = None,
```

Comment:
So for example, is `visibility_timeout` something we want here? While yes it does show up on a few APIs here, I don't think it is universal enough to omit i.e. naked `timeout`

Reply:
Look into type of `content` -- should probably be Bytes or something?

Reply:
Change default to `None`